### DISCUSSION TITLE
Fresh install cant find dependencies

### DISCUSSION THREAD

**Discussion by derrickbeining** [johnlindquist/kit#686] [Errors]
I freshly installed Kit.app and tried running the Dad jokes scripts and got this error:

```
kit is not defined
ReferenceError: kit is not defined
    at file:///Users/derrickbeining/.kenv/scripts/dad-joke.js
    at ModuleJob.run (node:internal/modules/esm/module_job:183:25)
    at async Loader.import (node:internal/modules/esm/loader:178:24)
    at async global.attemptImport (file:///Users/derrickbeining/.kit/api/kit.js:92:28)
    at async run (file:///Users/derrickbeining/.kit/core/utils.js:394:12)
    at async file:///Users/derrickbeining/.kit/cli/app-run.js
`

**Comment by johnlindquist** [comment]
The bundled dad joke script or a different one? 

Do other scripts work?

**Comment by imtaehyun** [comment]
Hi @johnlindquist 

I also installed it freshly, but I'm having same issue. 
bundled scripts are not working (dad joke, word-api, etc)

I tested from both ui and cli, but not working from both.

```
 ~  kit                                                                                                                                                                                            ✔  16:41:35
✔ What do you want to do? · run
✔ Which script do you want to run? · Word API
ReferenceError: kit is not defined
    at file:///Users/taehyuim/.kenv/scripts/word-api.js?uuid=c4746554-8599-4948-94b2-dc161a87b4a1:6:27
    at ModuleJob.run (node:internal/modules/esm/module_job:175:25)
    at async Loader.import (node:internal/modules/esm/loader:178:24)
    at async global.attemptImport (file:///Users/taehyuim/.kit/api/kit.js:102:28)
    at async run (file:///Users/taehyuim/.kit/core/utils.js:396:12)
    at async file:///Users/taehyuim/.kit/cli/run.js?uuid=37456ad5-4241-4f1e-b21b-f95beeb54784:4:1
```

**Comment by imtaehyun** [comment]
Yes, it does.
Also, I've added `~/.kit/bin` to the path in my .zshrc file.

2022년 2월 24일 (목) 오후 5:14, John Lindquist ***@***.***>님이 작성:

> Does your ~/.kit directory exist?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/johnlindquist/kit/discussions/686#discussioncomment-2247862>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AATJEPFXSUFZDRPXVZS7SXDU43JXDANCNFSM5L4UAJ3Q>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Thank you,

Best regards

Taehyun (Michael) Im

**Reply by johnlindquist** [reply]
Can you copy/paste the contents of the word-api script? Just curious if it's somehow pulling from an old source

**Reply by imtaehyun** [reply]
The last modified date is Feb, 2021. I remember that I installed initial version of kitapp and removed it, but `.kenv` folder wasn't deleted.
It's working properly after I cleaned up `.kit` and `.kenv` folders and reinstalled the app. 
Thanks!

**Reply by johnlindquist** [reply]
Thanks for following up. I really, really appreciate it!
